src/vs/code/node/cliProcessMain.ts

```diff
@@ -139,8 +139,18 @@ class Main {
 								return TPromise.wrapError(new Error(`${notFound(id)}\n${useId}`));
 							}
 
-							console.log(localize('foundExtension', "Found '{0}' in the marketplace.", id));
-							console.log(localize('installing', "Installing..."));
+							const installedExtension = installed.filter(e => getId(e.manifest) === id)[0];
```

Comment:
There is a check to early quit at line 117 if an extension is already installed. That check should be removed.

Reply:
Done.